xenial/2.3.6/Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM ubuntu:xenial
 
-MAINTAINER Daniel Alan Miller <dalanmiller@rethinkdb.com>
+MAINTAINER Gabor Boros <gabor.brs@gmail.com>
+ENV RETHINKDB_PACKAGE_VERSION 2.3.6~0xenial
 
-# Add the RethinkDB repository and public key
-# "RethinkDB Packaging <packaging@rethinkdb.com>" http://download.rethinkdb.com/apt/pubkey.gpg
-RUN apt-key adv --keyserver keys.gnupg.net --recv-keys 3B87619DF812A63A8C1005C30742918E5C8DA04A
```

Comment:
This was done on purpose in #1 and will properly cache bust when key changes happen whereas this new line (12) will be cached indefinitely even if the remote changes.

```Dockerfile
RUN apt-key adv --fetch-keys https://download.rethinkdb.com/apt/pubkey.gpg \
￼    && echo "deb https://download.rethinkdb.com/apt xenial main" > /etc/apt/sources.list.d/rethinkdb.list
```

See also https://github.com/docker-library/faq/#openpgp--gnupg-keys-and-verification and https://github.com/docker-library/official-images/tree/a4214453cb62b364ea8485ab5446d918fd4c304c#image-build